### ISSUE TITLE
ComboBox multi-select has invalid/missing aria attributes

### ISSUE THREAD

**Issue by anjais123** [microsoft/fluentui#15223]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/khali007/pen/yLYvOVR

###Prerequisites:
Add [accessibility insights extension for web](https://accessibilityinsights.io/docs/en/web/overview/).
#### Actual behavior:
When automated checks are executed for errors with accessibility insights for web extension, errors are reported as demonstrated below.

![image](https://user-images.githubusercontent.com/71385382/94198637-3510bd00-fed5-11ea-8381-9df12bf85a58.png)

![image](https://user-images.githubusercontent.com/71385382/94198866-946ecd00-fed5-11ea-9cc6-3f9adb4da768.png)

#### Expected behavior:

How to fix 
ARIA attribute is not allowed: aria-selected="true"
No error should be reported in accessibility insights for web.

**Comment by paulgildea** [comment]
@anjais123 Thanks for the feedback.

@smhigley FYI

@miroslavstastny - Try to reproduce with the latest website example.
